app/models/comment.rb

```diff
@@ -1,4 +1,6 @@
 class Comment < ApplicationRecord
   belongs_to :request
   belongs_to :author, polymorphic: true
+
+  validates :content, presence: true
```

Comment:
Exactly, but author and request are automatically set by our app not by user, so I don't think I need to validate them.
In this situation, how should I think about it?